docker-compose.yaml

```diff
@@ -24,11 +24,9 @@ services:
       - ./:/app
   db:
     image: postgres:14
-    container_name: integration-tests-prisma
```

Comment:
Are we using the `container_name`?

Reply:
No, i was just following Prisma's integration testing user guide